src/main/java/stream/flarebot/flarebot/commands/random/JumboCommand.java

```diff
@@ -0,0 +1,60 @@
+package stream.flarebot.flarebot.commands.random;
+
+import net.dv8tion.jda.core.entities.Emote;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import stream.flarebot.flarebot.FlareBot;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+public class JumboCommand implements Command {
+
+    @Override
+    public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message message, String[] args, Member member) {
+        if (args.length > 0) {
+            if (!message.getEmotes().isEmpty()) {
+                Emote e = message.getEmotes().get(0);
```

Comment:
You might want to check for if an emote is null someone chould still do `_jumbo thing <emote>` and it'd break

Reply:
Oh i see now. I don't know if we want to check for that though.

Reply:
Not particular